### ISSUE TITLE
train loss is negative

### ISSUE THREAD

**Issue by Purdandelion** [JiahuiYu/generative_inpainting#373]
hello,thanks for releaseing deepfillv2,I have loaded code and pretrained model, I use pretrained model to fine-tune model in my own dataset, I only change some training parameter. but the train loss is negative and decreaseing，I doubt it is due to my dataset，so I use place2 valitation set to fine-tune model，but the loss is also negative，I don’t konw why，can you tell me, is the code  wrong?
